orion_mcp.py

```
@@ -118,56 +119,103 @@ async def openshift_report_on(
+            "metric": metric,
+            "lookback": lookback,
+            "display": display if display.strip() else None,
+            "data": full_data,
+            "plot_info": "Image data follows JSON data"
+        }
+        try:
+            img_b64 = generate_multi_line_plot(series, metric, title_prefix=f"{config}: ")
+            combined_output = json.dumps(json_output, indent=2) + "\n\n[IMAGE_DATA_BASE64]\n" + img_b64.decode("utf-8")
+            return types.TextContent(type="text", text=combined_output)
+        except ValueError as e:
+            return types.TextContent(type="text", text=f"Error generating plot: {str(e)}\n\nJSON data:\n{json.dumps(json_output, indent=2)}")
```

Ruff (0.14.5)

Use explicit conversion flag

Replace with conversion flag

(RUF010)